regression.py

```diff
@@ -681,6 +681,14 @@ def testORCA_ORCA4_0_1_ttt_td_out(logfile):
     assert len(logfile.data.etsecs[0]) == 1
     assert numpy.isnan(logfile.data.etsecs[0][0][2])
 
+def testORCA_ORCA4_0_IrCl6_sp_out(logfile):
+    """Tests ECP and weird SCF printing."""
+    assert hasattr(logfile.data, 'scfvalues')
+    assert len(logfile.data.scfvalues) == 1
+    vals =  [0.000037800796, 0.00412549, 0.00014041]

```

Comment:
Actually, could you take the first one, where there's an overflow? I'm assuming `0.00000000000028.31276975  0.71923638` should become `[0.000000000000, 28.31276975, 0.71923638]`.